### PR TITLE
Update networking.md

### PR DESCRIPTION
**Pull request by danwen1**

### CODE REVIEW

File: v1.2.0/operator-guides/configure/networking.md [hunk 1]
```diff
@@ -1,44 +1,44 @@
 ---
-title: Configure networking
+title: 配置联网
```
Comment (JipeiWang) [inline v1.2.0/operator-guides/configure/networking.md]: 网络配置

File: v1.2.0/operator-guides/configure/networking.md [hunk 2]
```diff
@@ -1,44 +1,44 @@
...
 - Grafana
 
-# Access types
+# 访问类型
 
-You can configure Pulsar service with different types according to the scenarios, such as `ClusterIP`, `NodePort` and `LoadBalancer`, and use different access methods for different types.
+你可以根据场景配置不同类型的 Pulsar 服务，如 `ClusterIP`、`NodePort` 和 `LoadBalancer`，并对不同类型使用不同的访问方法。
 
-- `ClusterIP` exposes services through the internal IP of the cluster. When selecting this type of service, you can only access it within the cluster by the following methods:
+- `ClusterIP ` 通过集群的内部 IP 暴露服务。当选择这种类型的服务时，只能通过以下方法在集群内访问服务。
```
Comment (JipeiWang) [inline v1.2.0/operator-guides/configure/networking.md]:
```suggestion
- `ClusterIP ` 通过集群的内部 IP 暴露服务。当选择这种类型的服务时，只能通过以下方法在集群内访问服务：
```

File: v1.2.0/operator-guides/configure/networking.md [hunk 3]
```diff
@@ -1,44 +1,44 @@
...
-- `LoadBalancer`: clients connect to Pulsar proxies, Grafana, StreamNative Console using the Kubernetes provider’s load balancer. 
-- `NodePort`: If there is no LoadBalancer, you can choose to expose the service through NodePort. NodePort exposes services through the node's IP address and static port. You can access a `NodePort` service from outside of the cluster by requesting NodeIP + NodePort
+- `LoadBalancer`：客户端使用 Kubernetes 服务商的负载均衡器连接到 Pulsar proxy、Grafana、StreamNative 控制台。
+- `NodePort`：如果没有 LoadBalancer，可以选择通过 NodePort 暴露服务。NodePort 通过节点的 IP 地址和静态端口来暴露服务。可以通过 NodeIP + NodePort 从集群外部访问 `NodePort` 服务。
 
-Here is an example about obtaining the Pulsar services information by running the following command.
+以下是运行命令获得 Pulsar 服务信息的例子。
 
-**Example**
+**例子**
```
Comment (JipeiWang) [inline v1.2.0/operator-guides/configure/networking.md]:
```suggestion
**示例**
```

File: v1.2.0/operator-guides/configure/networking.md [hunk 4]
```diff
@@ -1,44 +1,44 @@
...
-# Access types
+# 访问类型
 
-You can configure Pulsar service with different types according to the scenarios, such as `ClusterIP`, `NodePort` and `LoadBalancer`, and use different access methods for different types.
+你可以根据场景配置不同类型的 Pulsar 服务，如 `ClusterIP`、`NodePort` 和 `LoadBalancer`，并对不同类型使用不同的访问方法。
 
-- `ClusterIP` exposes services through the internal IP of the cluster. When selecting this type of service, you can only access it within the cluster by the following methods:
+- `ClusterIP ` 通过集群的内部 IP 暴露服务。当选择这种类型的服务时，只能通过以下方法在集群内访问服务。
 
   - ClusterIP + ServicePort
```
Comment (JipeiWang) [inline v1.2.0/operator-guides/configure/networking.md]: 后面的 ServicePort 翻译为服务端口，前后需保持一致

**Review by JipeiWang — CHANGES_REQUESTED** [review]

File: v1.2.0/operator-guides/configure/networking.md (hunk 1 above)
Comment (Infomagic-ly) [reply v1.2.0/operator-guides/configure/networking.md]: done.

File: v1.2.0/operator-guides/configure/networking.md (hunk 2 above)
Comment (Infomagic-ly) [reply v1.2.0/operator-guides/configure/networking.md]: done.

File: v1.2.0/operator-guides/configure/networking.md (hunk 3 above)
Comment (Infomagic-ly) [reply v1.2.0/operator-guides/configure/networking.md]: done.

File: v1.2.0/operator-guides/configure/networking.md (hunk 4 above)
Comment (Infomagic-ly) [reply v1.2.0/operator-guides/configure/networking.md]: done.

File: v1.2.0/operator-guides/configure/networking.md [hunk 5]
```diff
@@ -1,44 +1,43 @@
...
 
-You can configure Pulsar service with different types according to the scenarios, such as `ClusterIP`, `NodePort` and `LoadBalancer`, and use different access methods for different types.
+你可以根据场景配置不同类型的 Pulsar 服务，如 `ClusterIP`、`NodePort` 和 `LoadBalancer`，并对不同类型使用不同的访问方法。
 
-- `ClusterIP` exposes services through the internal IP of the cluster. When selecting this type of service, you can only access it within the cluster by the following methods:
+- `ClusterIP ` 通过集群的内部 IP 暴露服务。当选择这种类型的服务时，只能通过以下方法在集群内访问服务：
+  - ClusterIP + 服务端口
+  - 服务域名 (`${serviceName}.${namespace}`) + 服务端口
+  
+- `LoadBalancer`：客户端使用 Kubernetes 服务商的负载均衡器连接到 Pulsar proxy、Grafana、StreamNative 控制台。
```
Comment (tuhaihe) [inline v1.2.0/operator-guides/configure/networking.md]: `Kubernetes provider`：不翻译

**Review by tuhaihe — CHANGES_REQUESTED** [review]

File: v1.2.0/operator-guides/configure/networking.md (hunk 5 above)
Comment (danwen1) [reply v1.2.0/operator-guides/configure/networking.md]: Updated.

**Review by tuhaihe — APPROVED** [review]
@tuhaihe Approved.

**Pull request merged**